guides/04_additional-features/15_internationalization.md

```diff
@@ -0,0 +1,58 @@
+Tags: internationalization, i18n, language
+Related spaces:
+
+# Internationalization (i18n)
+
+Gradio provides custom internationalization (i18n) support through the `I18n` class. This feature enables the translation of your application's text based on the user's browser locale and extends Gradio's built-in translation system with your custom translations.
```

Comment:
I would describe Gradio's built-in i18n a little bit more as well in case users are looking to understand that more. E.g. we automatically translate text that is part of the standard Gradio UI (e.g. "drop files here") in more than 40 languages, but if you want to add custom i18n...